contrib/mergeBenchmarkSets.py

```diff
@@ -22,6 +23,27 @@
 sys.dont_write_bytecode = True  # prevent creation of .pyc files
 
 
+def parse_args(argv):
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "resultsXML",
+        metavar="results-xml",
+        help="XML-file containing the verification results.",
+    )
+    parser.add_argument(
+        "witnessXML",
+        nargs="*",
+        metavar="witness-xml",
+        help="Any number of XML-files containing validation results.",
+    )
+    parser.add_argument(
+        "--no-overwrite-status-true",
```

Comment:
It is less confusing to have variables and fields in source code have "positive" identifiers. I.e., it would be better to have the (internal) option named `overwrite_status_true` (or simply `overwrite_status`?). The option for the user can still be negated where necessary, `argarse` has `store_false`.

Reply:
You're right, this option's name certainly is confusing.
Internally it is not used though, but stored as `isOverwrite` (which admittedly is a pretty bad name itself), and I didn't want to rename the option in case some scripts rely on it being called that.

Reply:
But this is a new option, so how can scripts rely on it?

Anyway, I didn't recommend renaming the option (what would a better name be?), but for renaming the internal field in which it is stored.

Note that `isOverwrite` would better be called `doOverwrite` but even better something like `overwriteStatus`.

Reply:
The option already existed before, I just added usage of argparse.
I renamed `isOverwrite` to `overwrite_status`.

Reply:
Ok, I understand. Then let's no mix this refactoring with interface changes.